WORKSPACE

```diff
@@ -121,3 +121,15 @@ http_archive(
         "https://github.com/jmhodges/bazel_gomock/archive/fde78c91cf1783cc1e33ba278922ba67a6ee2a84.tar.gz",
     ],
 )
+
+new_git_repository(
+    name = "com_github_cockroachdb_sqllogictest",
+    build_file_content = """
+filegroup(
+    name = "testfiles",
+    srcs = glob(["test/**/*.test"]),
+    visibility = ["//visibility:public"],
+)""",
+    commit = "96138842571462ed9a697bff590828d8f6356a2f",
```

Comment:
The non-Bazel build configuration is set up to always pull from `master`. We'll have to make sure that any changes that are pushed to the `master` branch of this repo are also reflected here.